CUSTOMIZATION.md

```diff
@@ -1,13 +1,16 @@
 # Customizing Your Workshop's Website
 
-You must change the values of `lesson_repo` and `lesson_site` in
-the `_config.yml` configuration file in the root directory of your
-workshop to point to the repository for the lesson and its GitHub
-pages URL respectively.  You should not need to modify any of the
-other values in `_config.yml`.
+## Configuration File
 
-Your workshop's `index.html` page must define the following values
-in its header:
+You must change the values of `lesson_repo` and `lesson_site`
```

Comment:
Replace 'values' with 'URLs'. 

Also, the commenting in the _config.yml is very user friendly :cookie: